eng/pipelines/dotnet-buildtools-prereqs-centos.yml

```
@@ -25,3 +25,4 @@ stages:
     internalProjectName: ${{ variables.internalProjectName }}
     publicProjectName: ${{ variables.publicProjectName }}
     linuxAmdBuildJobTimeout: 210
+    linuxArmBuildJobTimeout: 210
```

Looks like the timeout needs to be increased.
```suggestion
    linuxArmBuildJobTimeout: 270
```

---

This same timeout parameter needs to be added here:

https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/26b5ef43b338cd9f130ec4ee03994886947f8896/eng/pipelines/dotnet-buildtools-prereqs-eng.yml#L25-L30